motionplan/tpSpaceRRT.go

```diff
@@ -22,23 +22,21 @@ import (
 )
 
 const (
-	defaultAutoBB = 0.7 // Automatic bounding box on driveable area as a multiple of start-goal distance
+	defaultAutoBB = 0.8 // Automatic bounding box on driveable area as a multiple of start-goal distance
 	// Note: while fully holonomic planners can use the limits of the frame as implicit boundaries, with non-holonomic motion
 	// this is not the case, and the total workspace available to the planned frame is not directly related to the motion available
 	// from a single set of inputs.
 
 	// How much the bounding box of random points to sample increases in size with each algorithm iteration.
 	autoBBscale = 0.1
 
-	// whether to add intermediate waypoints.
-	defaultAddInt = true
```

Comment:
This, and most of the changes to defaults below, are to use values more appropriate for long PTGs. These were tuned back when a single trajectory rarely exceeded 2 meters.